src/main/java/cs2103/v15_1j/jimjim/TaskEvent.java

```diff
@@ -1,6 +1,6 @@
 package cs2103.v15_1j.jimjim;
 
-public abstract class TaskEvent {
+public class TaskEvent {
```

Comment:
GSON only works with concrete classes

On Friday, 4 March 2016, Tran Tien Dat <notifications@github.com> wrote:

> In src/main/java/cs2103/v15_1j/jimjim/TaskEvent.java
> <https://github.com/cs2103jan2016-v15-1j/main/pull/45#discussion_r54915938>
> :
>
> > @@ -1,6 +1,6 @@
> >  package cs2103.v15_1j.jimjim;
> >
> > -public abstract class TaskEvent {
> > +public class TaskEvent {
>
> why do you remove abstract? I think it's appropriate because this is just
> a wrapper class
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cs2103jan2016-v15-1j/main/pull/45/files#r54915938>.
>


-- 
Shaun Thium (Mr.)
*National University of Singapore*
B.Comp (Computer Science) | University Scholar's Programme